configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5-gpu.env

```diff
@@ -0,0 +1,17 @@
+GST_PIPELINE_LAUNCH=/home/pipeline-server/framework-pipelines/yolov5_pipeline/yolov5s.sh
+OCR_DEVICE=
+LOG_LEVEL=ERROR
+GST_DEBUG=0
+LOW_POWER=
+CPU_ONLY=
+AUTO_SCALE_FLEX_140=
+DECODE=! vaapidecodebin
```

Comment:
we need double quote around decode value

Reply:
actually you don't need the "! " in the front, the gst script already includes that